native/core/src/events.rs

```diff
@@ -207,11 +232,238 @@ impl<'de> serde::Deserialize<'de> for AnyActerEvent {
+                let event =
+                    smart_serde_json::from_str::<comments::SyncCommentUpdateEvent>(json.get())
+                        .map_err(D::Error::custom)?;
+                Ok(Self::CommentUpdate(event))
+            }
+
+            attachments::AttachmentEventContent::TYPE => {
+                let event =
+                    smart_serde_json::from_str::<attachments::SyncAttachmentEvent>(json.get())
+                        .map_err(D::Error::custom)?;
+                Ok(Self::Attachment(event))
+            }
+            attachments::AttachmentUpdateEventContent::TYPE => {
+                let event = smart_serde_json::from_str::<attachments::SyncAttachmentUpdateEvent>(
+                    json.get(),
+                )
+                .map_err(D::Error::custom)?;
+                Ok(Self::AttachmentUpdate(event))
+            }
+
+            rsvp::RsvpEventContent::TYPE => {
+                let event = smart_serde_json::from_str::<rsvp::SyncRsvpEvent>(json.get())
+                    .map_err(D::Error::custom)?;
+                Ok(Self::Rsvp(event))
+            }
+
+            read_receipt::ReadReceiptEventContent::TYPE => {
+                let event = ::matrix_sdk_base::ruma::exports::serde_json::from_str::<
+                    read_receipt::SyncReadReceiptEvent,
+                >(json.get())
+                .map_err(D::Error::custom)?;
+                Ok(Self::ReadReceipt(event))
+            }
+
+            reaction::ReactionEventContent::TYPE => {
+                let event = ::matrix_sdk_base::ruma::exports::serde_json::from_str::<
+                    reaction::SyncReactionEvent,
+                >(json.get())
+                .map_err(D::Error::custom)?;
+                Ok(Self::Reaction(event))
+            }
+
+            _ => Err(SerdeDeError::unknown_variant(
+                &ev_type,
+                &[
+                    calendar::CalendarEventEventContent::TYPE,
+                    calendar::CalendarEventUpdateEventContent::TYPE,
+                    pins::PinEventContent::TYPE,
+                    pins::PinUpdateEventContent::TYPE,
+                    news::NewsEntryEventContent::TYPE,
+                    news::NewsEntryUpdateEventContent::TYPE,
+                    tasks::TaskListEventContent::TYPE,
+                    tasks::TaskListUpdateEventContent::TYPE,
+                    tasks::TaskEventContent::TYPE,
+                    tasks::TaskUpdateEventContent::TYPE,
+                    tasks::TaskSelfAssignEventContent::TYPE,
+                    tasks::TaskSelfUnassignEventContent::TYPE,
+                    comments::CommentEventContent::TYPE,
+                    comments::CommentUpdateEventContent::TYPE,
+                    attachments::AttachmentEventContent::TYPE,
+                    attachments::AttachmentUpdateEventContent::TYPE,
+                    rsvp::RsvpEventContent::TYPE,
+                    reaction::ReactionEventContent::TYPE,
```

Comment:
`read_receipt::ReadReceiptEventContent::TYPE` seems to be missed.

Reply:
```suggestion
                    read_receipt::ReadReceiptEventContent::TYPE,
                    reaction::ReactionEventContent::TYPE,
```